<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baseList</class>
 <widget class="QDialog" name="baseList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>382</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comfortaa</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
   <item>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <family>Courier Prime</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tip">
     <property name="text">
      <string>tip</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="header">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>58</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="action2">
       <property name="text">
        <string>action2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="action1">
       <property name="text">
        <string>action1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
